elyra/pipeline/pipeline.py

```diff
@@ -102,6 +103,22 @@ def include_subdirectories(self):
     def env_vars(self):
         return self._env_vars
 
+    @property
+    def env_vars_as_dict(self) -> Dict:
+        """Operation stores environment variables in a list of name=value pairs, while
+           subprocess.run() requires a dictionary - so we must convert.  If no envs are
+           configured on the Operation, the existing env is returned, otherwise envs
+           configured on the Operation are overlayed on the existing env.
+        """
+        envs = os.environ.copy()
```

Comment:
We may want to revisit using the server's env in user requests at some point.  OK for now.